packetbeat/tests/system/test_0070_drda_simple.py

```diff
@@ -0,0 +1,13 @@
+from packetbeat import BaseTest
+
+class Test(BaseTest):
+
+    def test_simple(self):
+        self.render_config_template(
+            drda_ports=[1527],
+        )
+
+        self.run_packetbeat(pcap="drda/drda_simple.pcap", debug_selectors=["*"])
+        objs = self.read_output()
+
+        assert len(objs) == 21
```

Comment:
Would be good to check more things here from the individual objects, like that the method is set to the right value, etc.